### DISCUSSION TITLE
ERROR: opencv-4.5.5-r0 do_patch: Target directory '/buildarea1/jetson-wrlinux1/build/tmp-glibc/work/armv8a_tegra234-wrs-linux/opencv/4.5.5-r0/git/../contrib' not found, patchdir '../contrib' is incorrect in patch file '0011-fix-cuda-mem-leak-and-move-cuda-malloc-out-of-critic.patch'

### DISCUSSION THREAD

**Discussion by PugalendhiMurugan** [OE4T/meta-tegra-community#118] [Q&A]
Facing below error while compiling on the work-in-progress branch for the JetPack 6 Developer Preview
Initialising tasks: 100% |######################################################################################| Time: 0:00:09
Sstate summary: Wanted 2703 Local 0 Mirrors 0 Missed 2703 Current 1772 (0% match, 39% complete)
NOTE: Executing Tasks
ERROR: opencv-4.5.5-r0 do_patch: Target directory '/buildarea1/jetson-wrlinux1/build/tmp-glibc/work/armv8a_tegra234-wrs-linux/opencv/4.5.5-r0/git/../contrib' not found, patchdir '../contrib' is incorrect in patch file '0011-fix-cuda-mem-leak-and-move-cuda-malloc-out-of-critic.patch'
ERROR: Logfile of failure stored in: /buildarea1/jetson-wrlinux1/build/tmp-glibc/work/armv8a_tegra234-wrs-linux/opencv/4.5.5-r0/temp/log.do_patch.3621450
ERROR: Task (/buildarea1/jetson-wrlinux1/layers/meta-openembedded/meta-oe/recipes-support/opencv/opencv_4.5.5.bb:do_patch) failed with exit code '1'
Waiting for 1 running task to finish:
0: rust-native-1.59.0-r0 do_compile - 3m20s (pid 3620902)

**Comment by madisongh** [comment]
Make sure you layers are synchronized to the right branches.  The WIP branch tracks the `master` branch in OE-Core (last updated a couple of weeks back), and your other layers must do so also.  See [the branch in our demo distro repo](https://github.com/OE4T/tegra-demo-distro/tree/wip-l4t-r36.2.0-dp) for an example.

The version of OpenCV on `master` should be 4.9, not 4.5.5.  And I have no idea where the patch mentioned in that error message came from.
